### PR TITLE
Minor typos and grammarios

### PR DESCRIPTION
**Pull request by ghost**
:+1:

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -41,7 +41,7 @@ <h1 class="title">Scala Summer School</h1>
     <section id="about" class="js--section">
       <div class="row">
         <p class="long-text">
-          <font color="#e67e22">Thanks</font> to all participants of the 2018 Summer School workshop. It was a blast. All lecture, exercise and project materials can be found in our Github organization:
+          <font color="#e67e22">Thanks</font> to all participants of the 2018 Summer School workshop. It was a blast. All lectures, exercises and project materials available from our Github Org repository:
```
Comment (pheymann) [inline index.html:44]: Should it be `are available`?

**Comment by pheymann** [comment]
Hi @svatsan ,
thanks for fixing these typos and grammatical errors. I just left a small comment.

File: index.html (hunk 1 above)
Comment (ghost) [reply index.html:44]: IMHO, they are both correct. Saying `available from` indicates what follows that sentence. While `are available` indicates a statement. If it helps, please feel free to make it `are available` :-)

Comment (pheymann) [reply index.html:44]: 👍

**Pull request merged**